cachito/workers/config.py

```diff
@@ -2,15 +2,53 @@
 import logging
 import os
 import tempfile
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, cast
 
 import celery
 import kombu
+from celery.signals import worker_process_init
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger.thrift import JaegerExporter
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.instrumentation.celery import CeleryInstrumentor
+from opentelemetry.instrumentation.requests import RequestsInstrumentor
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
 
 from cachito.errors import ConfigError
 
 ARCHIVES_VOLUME = os.path.join(tempfile.gettempdir(), "cachito-archives")
 
+RequestsInstrumentor().instrument()
+
+
+@worker_process_init.connect(weak=False)
+def init_celery_tracing(*args, **kwargs):
+    """Initialize OTLP tracing, set the processor & endpoint."""
+    CeleryInstrumentor().instrument()
+    config = get_config()
+    if config.cachito_jaeger_exporter_endpoint:
```

Comment:
With the way how the Config classes currently work, this will fail with an AttributeError if:
- the config is not DevelopmentConfig
- the config is DevelopmentConfig but CACHITO_JAEGER_EXPORTER_ENDPOINT is not set
- the config is DevelopmentConfig, CACHITO_JAEGER_EXPORTER_ENDPOINT is set, but CACHITO_OTLP_EXPORTER_ENDPOINT is also set

(I think the fix needs to be done in the Config class, just illustrating the problem here)